routes/index.js

```diff
@@ -136,13 +142,20 @@ module.exports = function (passport) {
       .exists()
       .isMobilePhone('any').withMessage('Please enter valid phone number'),
 
-    check('signature').exists().withMessage('Please enter signature'),
+    check('signature', 'Electronic singature must be your \"Firstname Lastname\"')
+      .exists()
+      .trim()
+      .custom((value, { req }) => value === req.body.first + " " + req.body.last),
 
     check('ecSignature').exists(),
 
-    check('ecName').exists().withMessage('Please enter emergency contact name'),
+    check('ecName')
+      .exists().withMessage('Please enter emergency contact name')
+      .trim()
+      .not().matches(/[^0-9]*\d+?[^0-9]*/).withMessage('Emergency contact name cannot contain numbers'),
 
-    check('ecRel').exists().withMessage('Please enter emergency contact relationship'),
+    check('ecRel')
```

Comment:
y here you did not check if it is only string (no numbers )

Reply:
It totally makes sense that someone's name wouldn't consist of numbers, but I couldn't convince myself that there were no situations where a number might be needed in describing an emergency contact's relationship. For example, I imagined someone listing a "2nd cousin" or a "neighbor from 3 doors down" etc. as an emergency contact... and I didn't want to constrain them too much without asking.

Reply:
OK that makes sense you wanna me approve or you wanna let @danieleynis  take a look

Reply:
Normally, I would say approve... but since we're so close to the end I'd say the more eyes on it the better. I don'tn want to screw something up just before the presentation. :-P Lets wait for @danieleynis and/or @huynguyencs to have a look just in case.

Reply:
OK, and for your other pull request unfortunately, I can't review as I do not know exactly everything u did.

Reply:
Oh, we should definitely wait then... because this pull request includes that pull request's changes. I did the other one first and built this on top of it.